apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -65,15 +67,13 @@ function viewLinkAccessor(url: string, pool: Pool, intl: IntlShape) {
 }
 
 function dateAccessor(value: Maybe<string>, intl: IntlShape) {
-  return value ? (
-    <span>
-      {formatDate({
+  return value
+    ? formatDate({
         date: parseDateTimeUtc(value),
```

Comment:
I don't think this should be formatting a string.  If it wants a primitive sortable value it should probably be `parseDateTimeUtc(value).valueOf()`.  This is probalby the source of the date sorting I noticed on SearchRequestTable.tsx.

Reply:
Thanks, this seemed to work for me!

[fix table date accessors](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5649/commits/c53c69ad78d8a7681c2f207e0b685405414868f1)